README.md

```diff
@@ -15,83 +15,198 @@ npm run test
+		await page.evaluateOnNewDocument(FID);
+		await page.evaluateOnNewDocument(CLS);
+		await page.goto('http://localhost:8080/dropdown');
+		await page.waitForSelector('#dropdown');
+		await page.focus('#dropdown');
+		await page.keyboard.down('Enter');
 
-		const actualMount = Mount(filename, 'Changeable');
-		TestResults.addResult({component: 'Picker', type: 'Mount', actualValue: actualMount});
+		let actualFirstInput = await firstInputValue();
+		let actualCLS = await clsValue();
+
+		TestResults.addResult({component: component, type: 'FID', actualValue: Math.round((actualFirstInput + Number.EPSILON) * 1000) / 1000});
+		TestResults.addResult({component: component, type: 'CLS', actualValue: Math.round((actualCLS + Number.EPSILON) * 1000) / 1000});
+
+		expect(actualFirstInput).toBeLessThan(maxFID);
+		expect(actualCLS).toBeLessThan(maxCLS);
 	});
+
+	it('should have a good DCL, FCP and LCP', async () => {
+		const filename = getFileName(component);
+
+		let passContDCL = 0;
+		let passContFCP = 0;
+		let passContLCP = 0;
+		let avgDCL = 0;
+		let avgFCP = 0;
+		let avgLCP = 0;
+		for (let step = 0; step < stepNumber; step++) {
+			const dropdownPage = await testMultiple.newPage();
+
+			await dropdownPage.tracing.start({path: filename, screenshots: false});
+			await dropdownPage.goto('http://localhost:8080/dropdown');
+			await dropdownPage.waitForSelector('#dropdown');
+			await dropdownPage.waitForTimeout(200);
+
+			await dropdownPage.tracing.stop();
+
+			const {actualDCL, actualFCP, actualLCP} = PageLoadingMetrics(filename);
+			avgDCL = avgDCL + actualDCL;
+			if (actualDCL < maxDCL) {
+				passContDCL += 1;
+			}
+
+			avgFCP = avgFCP + actualFCP;
+			if (actualFCP < maxFCP) {
+				passContFCP += 1;
+			}
+
+			avgLCP = avgLCP + actualLCP;
+			if (actualLCP < maxLCP) {
+				passContLCP += 1;
+			}
+
+			await dropdownPage.close();
+		}
+		avgDCL = avgDCL / stepNumber;
+		avgFCP = avgFCP / stepNumber;
+		avgLCP = avgLCP / stepNumber;
+
+		TestResults.addResult({component: component, type: 'DCL', actualValue: Math.round((avgDCL + Number.EPSILON) * 1000) / 1000});
+		TestResults.addResult({component: component, type: 'FCP', actualValue: Math.round((avgFCP + Number.EPSILON) * 1000) / 1000});
+		TestResults.addResult({component: component, type: 'LCP', actualValue: Math.round((avgLCP + Number.EPSILON) * 1000) / 1000});
+
+		expect(passContDCL).toBeGreaterThan(passRatio * stepNumber);
+		expect(avgDCL).toBeLessThan(maxDCL);
+
+		expect(passContFCP).toBeGreaterThan(passRatio * stepNumber);
+		expect(avgFCP).toBeLessThan(maxFCP);
+
+		expect(passContLCP).toBeGreaterThan(passRatio * stepNumber);
+		expect(avgLCP).toBeLessThan(maxLCP);
+	});
+});
 ```
 
 ### Google Sheets
-We have the ability to send data to a Google Spreadsheet. If you wish to use this inclide an environment variable. 
+
+We have the ability to send data to a Google Spreadsheet. If you wish to use this includes an environment variable. 

```

Comment:
Can we still use this?

Reply:
Yes, please. I was not sure as well.

Reply:
I'm not sure.
Although the spreadsheet we operate does not exist, there is code that sends a POST request, and it seems to work.
Let's keep this part for now.